README.md

```diff
@@ -16,7 +16,7 @@ Welcome to our source speech repository about [flubber](https://github.com/veltm
  - Fabien JUIF [[Twitter](https://twitter.com/fabienjuif) | [Github](https://github.com/fabienjuif)]
 
 ## Deck
- - Soon
+ - [BDX.io Bordeaux, France - 2017 /en/](https://docs.google.com/presentation/d/16uJIRTJbvknvPyrH_nE-AmLdVBKw8hy6Rk6aAggrpYk/edit?usp=sharing)
```

Comment:
For PMP repo, we used the following format : [French version](...)
Do we need to share each version of the deck (for each time we played this talk :p) ?
In my opinion, the last version is suffisent

Reply:
I prefers to have one version per event in case we change the deck.
We can add a "latest version" if you want

Reply:
If someone visits the repo, he may be confused on don't know which version he should consult, don't you think ?